depends/packages/xtrans.mk

```diff
@@ -6,7 +6,7 @@ $(package)_sha256_hash=054d4ee3efd52508c753e9f7bc655ef185a29bd2850dd9e2fc2ccc335
 $(package)_dependencies=
 
 define $(package)_set_vars
-$(package)_config_opts_linux=--with-pic --disable-static
+$(package)_config_opts_linux=--with-pic --disable-shared
```

Comment:
Would be nice to do a rebase, so that changes already merged stop showing up here, no?

Reply:
That's the plan, Carl has just been afk for the past few days afaik. See discussion in [#bitcoin-builds](http://gnusha.org/bitcoin-builds/2019-06-23.log).
> 00:49 < dongcarl> I'm gunna clean up, upstream, document, then we can do last round of hashes!